contracts/Core/AssetManager.sol

```diff
@@ -259,79 +176,44 @@ contract AssetManager is ACL, AssetStorage, Constants, StateManager, IAssetManag
     //     }
     // }
 
-    function getJob(uint8 id)
-        external
-        view
-        override
-        returns (
-            uint8 selectorType,
-            uint8 weight,
-            int8 power,
-            string memory name,
-            string memory selector,
-            string memory url
-        )
-    {
-        require(jobs[id].assetType == uint8(AssetTypes.Job), "ID is not a job");
-
-        Structs.Job memory job = jobs[id];
-        return (job.selectorType, job.weight, job.power, job.name, job.selector, job.url);
-    }
-
-    function getCollection(uint8 id)
-        external
-        view
-        returns (
-            bool active,
-            int8 power,
-            uint8[] memory jobIDs,
-            uint32 aggregationMethod,
-            string memory name
-        )
-    {
-        require(collections[id].assetType == uint8(AssetTypes.Collection), "ID is not a collection");
-
-        return (
-            collections[id].active,
-            collections[id].power,
-            collections[id].jobIDs,
-            collections[id].aggregationMethod,
-            collections[id].name
-        );
-    }
-
-    function getAssetType(uint8 id) external view returns (uint8) {
+    function getAsset(uint8 id) external view returns (Structs.Job memory job, Structs.Collection memory collection) {
```

Comment:
Hey here, 
can we add condition, that only return whats not null, 
returning both, and having one of them null, may confuse clients